### PR TITLE
feat: added cart page

### PR DESCRIPTION
**Pull request by partha8**
Hey everyone, here's the cart page. The CSS for cart page is a little big and there's a chance that I might have missed something. Please share your reviews by going through the code. Thank you!

### CODE REVIEW

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+/* item block qty */
+
+.ib-qty-total-price {
+  display: flex;
+  gap: 1rem;
+  margin-left: auto;
+}
+
+.ib-qty {
+  /* width: 20%; */
```
Comment (sainath7878) [inline css/cart.css]: you can remove commented out code.

File: css/cart.css [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+  width: 85%;
+  max-width: 1024px;
+  margin: 0 auto;
+  padding: 3rem 0;
+}
+
+.btn {
+  display: inline-block;
+  background: var(--primary-color);
+  color: var(--clr-grey-10);
```
Comment (sainath7878) [inline css/cart.css]: Naming shouldn't be name of the color, rather it should be for what purpose we are using.

Comment (partha8) [reply css/cart.css]: I agree, but the grey color that I am using has different shades, and keeping a track of the shades is feasible this way. I have used that naming convention in my components library where I have three sections, Darker shades of grey, medium dark, and light grey. The numbers denote where it lies on this scale. The number ranges from 900 to 10, 10 being the lightest and 900 being the darkest.

I have opted for this method simply to have a variety and use it accordingly.

But I will certainly make changes to the color variables to better suit this project. Thank you @sainath7878 for reviewing.

**Review by foolhardy21 — APPROVED** [review]
The site and code looks good. One suggestion is that you could take some of the properties like flex, margins or paddings and add them as utils in you library and use it from that stylesheet.

**Review by ParthParolekar — APPROVED** [review]
Looks good to me!

File: css/cart.css (hunk 1 above)
Comment (partha8) [reply css/cart.css]: Removed them, thank you

File: css/cart.css (hunk 2 above)
Comment (partha8) [reply css/cart.css]: I have updated the color variables, please have a look at it.

**Comment by partha8** [comment]
> The site and code look good. One suggestion is that you could take some of the properties like flex, margins or paddings and add them as utils in your library and use them from that stylesheet.

I did think of this, but I wanted to keep the HTML code clean, and adding the properties in a particular class makes sense to me. I took this advice after Vikram mentioned utils' usage in a lecture. But I will keep this in mind. Thank you for the advice @foolhardy21!

**Comment by foolhardy21** [comment]
> > The site and code look good. One suggestion is that you could take some of the properties like flex, margins or paddings and add them as utils in your library and use them from that stylesheet.
> 
> I did think of this, but I wanted to keep the HTML code clean, and adding the properties in a particular style makes sense to me. I took this advice after Vikram mentioned utils' usage in a lecture. But I will keep this in mind. Thank you for the advice @foolhardy21!

Yeah, clean html is also a good point.👍🏾

**Review by abhisheksahani-nova — APPROVED** [review]
Everything looks fine

**Review by sainath7878 — COMMENTED** [review]
Looks good

**Pull request merged**